### PR TITLE
Dropdown: add IS_SELECTED_CLASS to option.selected (#331)

### PR DESCRIPTION
**Pull request by sevenam**
Proposed fix to #331

### CODE REVIEW

File: src/components/Dropdown/Dropdown.ts [hunk 1]
```diff
@@ -83,6 +83,9 @@ namespace fabric {
         if (option.disabled) {
           newItem.classList.add(IS_DISABLED_CLASS);
         }
+		if(option.selected) {
```
Comment (Jahnp) [inline src/components/Dropdown/Dropdown.ts]: Indentation/spacing is a little off here. Could you make this match the formatting used on lines 83 - 85?

**Comment by Jahnp** [comment]
Thanks @sevenam, looks great! I left one comment about spacing; once that's fixed I'm happy to approved and merge.

**Comment by sevenam** [comment]
Sorry about that. Spacing and linting fixed now and Travis seems to be happy :)

**Comment by Jahnp** [comment]
Approved

Thanks @sevenam!

**Comment by Jahnp** [comment]
Approved

**Comment by Jahnp** [comment]
Sorry for the delay here @sevenam, had to cleanup our approval processes a little :) Thanks again for the contribution!

**Pull request merged**